tools/operator-courier/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM registry.access.redhat.com/ubi8/python-36
 
-RUN pip3 install operator-courier
+RUN pip3 install --upgrade operator-courier==2.1.9
```

Comment:
this is not required anymore. It was a temporary workaround

Reply:
Tnx. removed it